src/verifiers/documentStatus/tokenRegistry/ethereumTokenRegistryStatus.ts

```diff
@@ -57,15 +56,24 @@ export const getTokenRegistry = (
   }
 };
 
+const isNonExistentToken = (error: any) => {
+  const message: string | undefined = error.body?.error?.message;
+  if (!message) return false;
+  return message.includes("owner query for nonexistent token");
+};
+const isMissingTokenRegistry = (error: any) => {
+  return (
+    !error.reason &&
+    error.method?.toLowerCase() === "ownerOf(uint256)".toLowerCase() &&
+    error.code === errors.CALL_EXCEPTION
+  );
+};
 const decodeError = (error: any) => {
   const reason = error.reason && Array.isArray(error.reason) ? error.reason[0] : error.reason ?? "";
   switch (true) {
-    case reason.toLowerCase() === "ERC721: owner query for nonexistent token".toLowerCase() &&
-      error.code === errors.CALL_EXCEPTION:
+    case isNonExistentToken(error):
```

Comment:
Hummmm that means that we will need to upgrade to this new version for this specific fix

I guess it's ok it's a very specific case, at worst it will throw a different error

Reply:
I honestly dont know a better way to go about this. I hate that there is so much coupling with the infura's error message...

Reply:
It's ok for this MR, I mainly mean for our different tools, it's not a big deal I htink, but sth to keep in mind